packages/patternfly-4/react-catalog-view-extension/less/catalog-item.less

```diff
@@ -0,0 +1,25 @@
+.catalog-item-header-pf {
+  display: flex;
+  align-items: center;
+}
+
+.catalog-item-header-pf-icon {
+  font-size: 60px;
+  max-height: 60px;
+  width: 60px;
+}
+
+.catalog-item-header-pf-text {
+  margin-left: 20px;
+}
+
+.catalog-item-header-pf-title {
+  font-weight: 400;
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+.catalog-item-header-pf-subtitle {
+  color: @color-pf-black-500;
+  margin-bottom: 0;
+}
```

Comment:
I don't believe there is a need to support Less anymore.

Reply:
I would be thrilled to remove it. I'll work on that later today.

Reply:
I removed the less support.